gui/WorldView.cxx

```diff
@@ -538,6 +553,80 @@ void WorldView::setGroundPlaneVisible(bool state)
 }
 
 //-----------------------------------------------------------------------------
+
```

Comment:
There are several style errors. This blank line should not be here. The next function is missing its separator line. Some lines are over length. Some spaces are missing after commas. Some type modifiers are incorrectly aligned. I will probably submit a PR to fix these rather than attempt to enumerate them.